### ISSUE TITLE
Add who-tag test set to the corpus

### ISSUE THREAD

**Issue by MansMeg** [welfare-state-analytics/riksdagen-corpus#187]
We want to include the quality of the who tags/connections to the corpus. This is because of two reasons. First, to use as a training set stored with the data, and secondly for continuous testing of data quality.

We need to
- [ ] Add the data to the corpus using the structure suggested by @ninpnin, make it explicit that this is a testset
- [ ] Automatically estimate the error rate based on the test set
- [ ] Add a unit test that checks that the error rate does not go down

**Comment by ninpnin** [comment]
I suggest this is done as part of the manual annotation round we conduct after v0.4.6 is ready.

**Comment by BobBorges** [comment]
Is this issue moot after #319?

**Comment by MansMeg** [comment]
Yes. I think so. @ninpnin ?

**Comment by ninpnin** [comment]
Done.
